cefpanda/__init__.py

```diff
@@ -82,13 +82,22 @@ def OnLoadError(self, **kwargs): #pylint: disable=invalid-name
 class CEFPanda(DirectObject):
     def __init__(self, transparent=True, size=None, parent=None):
         super().__init__()
-        cef_mod_dir = cefpython.GetModuleDirectory()
+        # Common application settings
         app_settings = {
             "windowless_rendering_enabled": True,
-            "locales_dir_path": os.path.join(cef_mod_dir, 'locales'),
-            "resources_dir_path": cefpython.GetModuleDirectory(),
-            "browser_subprocess_path": os.path.join(cef_mod_dir, 'subprocess'),
         }
+        cef_mod_dir_root = cefpython.GetModuleDirectory()
+        if sys.platform == "darwin":
+            app_settings['external_message_pump'] = True
+            # Detect if we are running in a bundled app, and if so fix the path
+            # to the framework resources
+            if p3d.ExecutionEnvironment.getEnvironmentVariable("MAIN_DIR").startswith(cef_mod_dir_root):
+                app_settings['browser_subprocess_path'] = os.path.normpath(os.path.join(cef_mod_dir_root, '../Frameworks/subprocess'))
```

Comment:
To avoid long line-length errors, we can change these lines to this form:
```
app_settings['setting'] = os.path.normpath(
    os.path.join(...)
)
```

Reply:
Ok, done